src/components/VmDialog/index.js

```diff
@@ -356,6 +361,7 @@ class VmDialog extends React.Component {
             onChange={this.onChangeTemplate}
             value={template ? template.get('id') : ''}
             data={sortedTemplates}
+            fieldHelp={<FieldHelp title='Template' content='Base configuration the virtual machine will be created from. Please customize if needed.' />}
```

Comment:
I would add that also the disks are there. Something like:
"Contains the configuration and disks which will be used to create this virtual machine. Please customize as needed"